src/Magnum/Platform/EmscriptenApplication.h

```diff
@@ -1486,7 +1486,8 @@ code. See
 @code{.cpp}
 int main(int argc, char** argv) {
     className app({argc, argv});
-    return app.exec();
+    app.exec();
+    return 0;
```

Comment:
Nope, app.exec() returns `0` now (see https://github.com/mosra/magnum/pull/300/commits/2383159ce4269901af3e86924270f9402a06b4a8). Actually, the macro should be updated to `return app.exec()` as well.

Reply:
Ah, I see